libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/TunnelEntranceAlert.kt

```diff
@@ -22,13 +24,39 @@ class TunnelEntranceAlert private constructor(
     /**
      * Transform this object into a builder to mutate the values.
      */
-    fun toBuilder(): Builder = Builder(coordinate, distance).alertGeometry(alertGeometry)
+    fun toBuilder(): Builder = Builder(coordinate, distance)
+        .alertGeometry(alertGeometry)
+        .info(info)
+
+    /**
+     * Indicates whether some other object is "equal to" this one.
+     */
+    override fun equals(other: Any?): Boolean {
+        if (this === other) return true
+        if (javaClass != other?.javaClass) return false
+        if (!super.equals(other)) return false
+
+        other as TunnelEntranceAlert
+
+        if (info != other.info) return false
```

Comment:
interesting the other properties are not part of equals, why is that?

Reply:
Isn't it? https://github.com/mapbox/mapbox-navigation-android/blob/b055dbe091cce0c310d3f0077cdbc35e0668fdfb/libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/TunnelEntranceAlert.kt#L37

Reply:
oh yep, that inheritance